src/test/java/Homework17Test.java

```diff
@@ -10,14 +10,23 @@
 public class Homework17Test extends BaseTest {
 
 @Test
-public void validLoginEmailPasswordTest(){
+public void validLoginEmailPassword(){
     openLoginURL();
     enterEmail("skyeman75@gmail.com");
     enterPassword("te$t$tudent");
     clickButton();
     WebElement avatarIcon =driver.findElement(By.cssSelector("[alt='Avatar of student']"));
     Assert.assertTrue(avatarIcon.isDisplayed());
 }
-
-
+@Test
+public void addSongToPlayList(){
+    WebElement allSongs = driver.findElement(By.cssSelector("a[href*='#!/songs']"));
+    allSongs.click();
+    WebElement songSelect = driver.findElement(By.cssSelector("td.title"));
+    songSelect.click();
+    WebElement addSongButton = driver.findElement(By.cssSelector(".btn-add-to"));
+    addSongButton.click();
+    WebElement addSong =driver.findElement(By.cssSelector("a[href*='#!/playlist/20795']"));
+    addSong.click();
```

Comment:
**Please, use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

        Assert.assertEquals(newName, getUsersNameOnHeader());

        Assert.assertEquals(newName, getUsernameProfileField());